src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -363,8 +363,8 @@ void Pooling::prepareParams() {
                                                key.effective_dilation,
                                                key.data_pad_end);
         DnnlDesriptor desc{desc_ptr};
-        pooling_v2_forward::primitive_desc prim_desc;
         primitive_desc_iterator itpd = desc.createPrimitiveDescriptorIterator(engine, key.attr);
+        pooling_v2_forward::primitive_desc prim_desc = itpd.get(true);
```

Comment:
Since you call `get` with `allow_empty = true`, it does make sense to check the primitve_descriptor is not empty (cast to bool for example) and return `nullptr` so the result will be correctly processed and exception will be thrown.

Reply:
We can't just cast the primitve_descriptor to bool because it returns true even for empty pd. So my suggestion is to set allow_empty to false: in this case we get an exception from oneDNN and don't return an empty pd from `builder`.